modules/ui/src/main/scala/lucuma/ui/optics/ValidateNec.scala

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.optics
+
+import cats.syntax.all._
+import cats.data.Validated
+import lucuma.core.optics.Format
+import monocle.Iso
+import monocle.Prism
+import cats.data.NonEmptyChain
+import cats.data.ValidatedNec
+
+object ValidateNec {
```

Comment:
I wonder if these names will be too confusing when we also have `cats.data.ValidatedNec`

Reply:
I wondered the same thing. Suggestions welcome.

Maybe `Validate*` => `Validator*` ?

Reply:
`ValidFormat*`?

Reply:
Isn't naming the hardest problem in computer science

Reply:
Went with `ValidFormat*`